File: runtime/CSlib/bucs2.cs
namespace bigloo 
{
  public sealed class bucs2: obj 
  {
    public readonly char value;

    public bucs2( char  value ) 
    {
      this.value= value;
    }

    public override void write( output_port  p ) 
    {
      p.write( "#u" + value );
    }
  }
}

File: runtime/Jlib/bucs2.java
package bigloo;

public class bucs2 extends obj
{
  public final char value;

  public bucs2( final char  value )
  {
    this.value= value;
  }

  public void write( final output_port  p )
  {
    p.write( "#u" + value );
  }
}
